test/e2e/storage/flexvolume_mounted_volume_resize.go

```diff
@@ -125,10 +122,10 @@ var _ = utils.SIGDescribe("Mounted flexvolume expand[Slow]", func() {
 
 	ginkgo.It("Should verify mounted flex volumes can be resized", func() {
 		driver := "dummy-attachable"
-		nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
-		node := nodeList.Items[0]
+		node, err := e2enode.GetRandomReadySchedulableNode(f.ClientSet)
+		framework.ExpectNoError(err)
```

Comment:
I guess I got the point why the pull-kubernetes-e2e-gce-storage-slow job was failed.
At the original line 71, `nodeName` is always the first node of the list.
And the label is added to the first node at original line 82.
In addition, flexvolume is installed into the node which is selected here.

After applying this change, both nodes of the original line 71 and 128 are selected randomly.
So if they are different, the test is failed.
That means the original code always selects the same node at these lines.

So it is better to change here and line 69 like
```
nodeList, err := e2enode.GetReadySchedulableNodes(f.ClientSet)
framework.ExpectNoError(err)
node := nodeList.Items[0]
```

Reply:
Aha! I was originally thinking it must be something to do with the introduction of node randomization, but I missed that there was dependency between the two `GetReadySchedulableNodes` calls in this file.

I kept the use of `GetRandomReadySchedulableNode` in the BeforeEach and just made it save the selected node and use the same one again in the test.

Reply:
Thanks for updating, that makes sense :-)